hack/build/config.sh

```diff
@@ -88,11 +88,8 @@ function parseTestOpts() {
 }
 
 function getTestPullPolicy() {
-    local pp
+    local pp=$PULL_POLICY
```

Comment:
So this function explains a lot, this tells me in Open Shift the pull policy should be always because latest doesn't always pull.. Of course with external you can't tell what it is, so maybe we should default external to always as well.
But I think all of this is too complicated. just return $PULL_POLICY and default it to IfNotPresent if not defined.

Reply:
So why not remove this func altogether? btw, won't it break CDI OpenShift ci, as it will use IfNotPresent instead of Always?

Reply:
There is no cdi openshift ci. This function was when we have okd lanes in our ci, which was a long time ago.

Reply:
ok, fixed.